ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -111,13 +96,13 @@ public typealias MSPillButton = PillButton
 /// An `PillButton` is a button in the shape of a pill that can have two states: on (Selected) and off (not selected)
 @objc(MSFPillButton)
 open class PillButton: UIButton {
-    private struct Constants {
-        static let bottomInset: CGFloat = 6.0
-        static let cornerRadius: CGFloat = 16.0
-        static let font: UIFont = Fonts.button4
-        static let horizontalInset: CGFloat = 16.0
-        static let topInset: CGFloat = 6.0
-    }
+
+    @objc public static let outlineStyleBackgroundColor = UIColor(light: Colors.surfaceTertiary, dark: Colors.surfaceSecondary)
+    @objc public static let outlineStyleTitleColor = UIColor(light: Colors.textSecondary, dark: Colors.textPrimary)
+    @objc public static let outlineStyleTitleSelectedColor = UIColor(light: Colors.textOnAccent, dark: Colors.textDominant)
+    @objc public static let filledStyleTitleColor = UIColor(light: Colors.textOnAccent, dark: outlineStyleTitleColor)
+
+    @objc public static let cornerRadius: CGFloat = 16.0
```

Comment:
Do we actually need all of these? Is there an simpler approach of modifications needed so we can just use PillButton instead of needing all these constants to recreate it?

Reply:
not sure I think cornerRadius should be exposed

Reply:
Since these are public they should probably have some documentation, which in this case seems pretty useful to explain why these are exposed (to allow clients to build custom PillButtonBars with similar appearance)